interface-definitions/interfaces-wireguard.xml.in

```diff
@@ -23,6 +23,7 @@
           #include <include/port-number.xml.i>
           #include <include/interface-mtu-68-16000.xml.i>
           #include <include/interface-ipv6-options.xml.i>
+          #include <include/interface-mirror.xml.i>
```

Comment:
Have you tested mirroring a WireGuard interface?

Reply:
As long as it can be set to tc, it should be possible, but there is no test for now, or delete it.